scripts/reborn_webui_v2_live_qa/run_live_qa.py

```
@@ -7262,6 +7342,40 @@ async def case_qa_10g_slack_last_message_sent_global(
+        details["search_index_readiness"] = readiness
+        if not readiness.get("ready"):
+            reason = (
+                "Slack search did not index the workspace-global last-sent "
+                f"marker within {int(SLACK_SEARCH_INDEX_READINESS_TIMEOUT_SECONDS)}s "
+                f"(attempts={readiness.get('attempts')}, "
+                f"error={readiness.get('error')!r}) — external search-index "
+                "lag, not an agent regression"
+            )
+            result = _result(
+                case_name,
+                False,
```

Ruff (0.15.21)

[warning] Boolean positional value in function call

(FBT003)